test/fixtures/wallet-payment.fixtures.js

```diff
@@ -21,6 +21,21 @@ const fixtures = {
     }
     return data
   },
+  stripeGoogleAuthRequestDetails: (ops = {}) => {
+    return {
+      payment_info: {
+        last_digits_card_number: ops.lastDigitsCardNumber !== undefined ? ops.lastDigitsCardNumber : successfulLastDigitsCardNumber,
+        brand: 'visa',
+        cardholder_name: 'Name',
+        email: 'name@example.com',
+        worldpay_3ds_flex_ddc_result: null,
```

Comment:
As its stripe - do we need to define this?

Reply:
we don't, it'll default to null on connectors end if it's not specifically defined here anyway so can be removed.